MediaBrowser.Controller/Entities/Folder.cs

```diff
@@ -593,8 +593,7 @@ void UpdateProgress()
                 progress.Report(childrenProgress.Average());
             }
 
-            var fanoutConcurrency = ConfigurationManager.Configuration.LibraryScanFanoutConcurrency;
-            var parallelism = fanoutConcurrency > 0 ? fanoutConcurrency : 2 * Environment.ProcessorCount;
+            var parallelism = Math.Max(ConfigurationManager.Configuration.LibraryScanFanoutConcurrency, 1);
```

Comment:
This completely removes the ability to set the value to be based on processor count... is that desired? The validation and descriptions in web will need updated if so. 🙃

Reply:
Our database can't handle that much concurrency, so I think having it based on processor count won't actually work